.azure-pipelines/test_plan.py

```diff
@@ -168,6 +168,7 @@ def __init__(self, url, frontend_url, client_id=None):
         if self.client_id:
             self.with_auth = True
             self.get_token()
+        self.init_time = datetime.now()
```

Comment:
suggest `token_init_time`

Reply:
If we merge the logic between login and getToken, no need the parameter here

Reply:
Indeed this is not token init time. It's the time when the `TestPlanManager` instance created. Use `token_init_time` would be mis-leading.

Reply:
Updated to `last_login_time`.
Good catch about line 170. Purpose of line 170 is to initialize some instance variables like `self._token`, `self._token_expires_on`. Initialization of `self.last_login_time` should be executed before calling `self.get_token()`. Updated.